modules/yardstick/config/benchmark-cache-load.properties

```diff
@@ -74,13 +72,21 @@ t=64
 # Sync mode.
 sm=PRIMARY_SYNC
 
-# Preload amount 
+# Preload amount
 pa=500000
 
 # Key range
 r=1000000
 
+# Batch size
+
+bs=50
+
+# Allow Operation
+ops="PUT PUT_ALL GET GET_ALL INVOKE INVOKE_ALL REMOVE REMOVE_ALL PUT_IF_ABSENT REPLACE SCAN_QUERY SQL_QUERY CONTINUOUS_QUERY"
```

Comment:
I would better list operation in the following way "--allowedLoadTestOperation=PUT, PUT_ALL, GET". This way it looks more natural when we need to specify multiple values for a parameter.